src/Quizzer.API.DAL/Seeds/Storage.cs

```diff
@@ -0,0 +1,559 @@
+                Score = 3.3,
+                QuizId = QuizGuids[1],
+                UserId = UserGuids[1]
+            });
+        }
+
+        private void SeedUserAnswers() {
+            // user 0, quiz 0
+            UserAnswers.Add(new UserAnswerEntity {
+                Id = UserAnswerGuids[0],
+                AnswerTime = DateTime.Parse("15.01.2023 19:34", System.Globalization.CultureInfo.CreateSpecificCulture("fr-FR")),
+                UserId = UserGuids[0],
+                QuestionId = QuestionGuids[0]
+            });
+
+            UserAnswers.Add(new UserAnswerEntity {
+                Id = UserAnswerGuids[1],
+                AnswerTime = DateTime.Parse("15.01.2023 19:34", System.Globalization.CultureInfo.CreateSpecificCulture("fr-FR")),
+                UserId = UserGuids[0],
+                QuestionId = QuestionGuids[1]
+            });
+
+            UserAnswers.Add(new UserAnswerEntity {
+                Id = UserAnswerGuids[2],
+                AnswerTime = DateTime.Parse("15.01.2023 19:34", System.Globalization.CultureInfo.CreateSpecificCulture("fr-FR")),
+                UserId = UserGuids[0],
+                QuestionId = QuestionGuids[2]
+            });
+
+
+            // user 1, quiz 0
+            UserAnswers.Add(new UserAnswerEntity {
+                Id = UserAnswerGuids[3],
+                AnswerTime = DateTime.Parse("16.01.2023 19:34", System.Globalization.CultureInfo.CreateSpecificCulture("fr-FR")),
+                UserId = UserGuids[1],
+                QuestionId = QuestionGuids[0]
+            });
+
+            UserAnswers.Add(new UserAnswerEntity {
+                Id = UserAnswerGuids[4],
+                AnswerTime = DateTime.Parse("16.01.2023 10:40", System.Globalization.CultureInfo.CreateSpecificCulture("fr-FR")),
+                UserId = UserGuids[1],
+                QuestionId = QuestionGuids[2]
+            });
+
+            // user 1, quiz 1
+            UserAnswers.Add(new UserAnswerEntity {
+                Id = UserAnswerGuids[5],
+                AnswerTime = DateTime.Parse("18.01.2023 10:42", System.Globalization.CultureInfo.CreateSpecificCulture("fr-FR")),
+                UserId = UserGuids[1],
+                QuestionId = QuestionGuids[3]
+            });
+
+            UserAnswers.Add(new UserAnswerEntity {
+                Id = UserAnswerGuids[6],
+                AnswerTime = DateTime.Parse("18.01.2023 10:43", System.Globalization.CultureInfo.CreateSpecificCulture("fr-FR")),
+                UserId = UserGuids[1],
+                QuestionId = QuestionGuids[4]
+            });
+
+            UserAnswers.Add(new UserAnswerEntity {
+                Id = UserAnswerGuids[7],
+                AnswerTime = DateTime.Parse("18.01.2023 19:48", System.Globalization.CultureInfo.CreateSpecificCulture("fr-FR")),
+                UserId = UserGuids[1],
+                QuestionId = QuestionGuids[5]
+            });
+
+            UserAnswers.Add(new UserAnswerEntity {
```

Comment:
Selected Answers don't need to be seeded here?

Reply:
No, it does not.